### PR TITLE
FIX #7781 - Now we can compile SuiteP only one color_scheme

### LINKED ISSUE #7781: RoboTask buildSuitep not using color_scheme argument.
#### Issue

If you want to build only one and only one `color_scheme` your are supposed to run:

```bash
$ ./vendor/bin/robo build:suite-p --color_scheme Dawn
```

But this RoboTask builds all color_schemes:

```bash
  Compile SuiteP Theme (SASS)
PHP Notice:  Array to string conversion in /home/jose/trabajos/SuiteCRM/lib/Robo/Plugin/Commands/BuildCommands.php on line 63
PHP Stack trace:
PHP   1. {main}() /home/jose/trabajos/SuiteCRM/vendor/consolidation/robo/robo:0
PHP   2. Robo\Runner->execute() /home/jose/trabajos/SuiteCRM/vendor/consolidation/robo/robo:48
PHP   3. Robo\Runner->run() /home/jose/trabajos/SuiteCRM/vendor/consolidation/robo/src/Runner.php:145
PHP   4. Robo\Application->run() /home/jose/trabajos/SuiteCRM/vendor/consolidation/robo/src/Runner.php:220
PHP   5. Robo\Application->doRun() /home/jose/trabajos/SuiteCRM/vendor/symfony/console/Application.php:148
PHP   6. Robo\Application->doRunCommand() /home/jose/trabajos/SuiteCRM/vendor/symfony/console/Application.php:255
PHP   7. Consolidation\AnnotatedCommand\AnnotatedCommand->run() /home/jose/trabajos/SuiteCRM/vendor/symfony/console/Application.php:978
PHP   8. Consolidation\AnnotatedCommand\AnnotatedCommand->execute() /home/jose/trabajos/SuiteCRM/vendor/symfony/console/Command/Command.php:255
PHP   9. Consolidation\AnnotatedCommand\CommandProcessor->process() /home/jose/trabajos/SuiteCRM/vendor/consolidation/annotated-command/src/AnnotatedCommand.php:302
PHP  10. Consolidation\AnnotatedCommand\CommandProcessor->validateRunAndAlter() /home/jose/trabajos/SuiteCRM/vendor/consolidation/annotated-command/src/CommandProcessor.php:176
PHP  11. Consolidation\AnnotatedCommand\CommandProcessor->runCommandCallback() /home/jose/trabajos/SuiteCRM/vendor/consolidation/annotated-command/src/CommandProcessor.php:212
PHP  12. call_user_func_array:{/home/jose/trabajos/SuiteCRM/vendor/consolidation/annotated-command/src/CommandProcessor.php:257}() /home/jose/trabajos/SuiteCRM/vendor/consolidation/annotated-command/src/CommandProcessor.php:257
PHP  13. SuiteCRM\Robo\Plugin\Commands\BuildCommands->buildSuiteP() /home/jose/trabajos/SuiteCRM/vendor/consolidation/annotated-command/src/CommandProcessor.php:257
➜  Found style.css for Dusk, Removing
➜  Found style.scss for Dusk, Compiling
 [Exec] Running ./vendor/bin/pscss -f compressed themes/SuiteP/css/Dusk/style.scss > themes/SuiteP/css/Dusk/style.css
 [Exec] Done in 30.76s
➜  Found style.css for Night, Removing
➜  Found style.scss for Night, Compiling
 [Exec] Running ./vendor/bin/pscss -f compressed themes/SuiteP/css/Night/style.scss > themes/SuiteP/css/Night/style.css
 [Exec] Done in 30.725s
➜  Found style.css for Dawn, Removing
➜  Found style.scss for Dawn, Compiling
 [Exec] Running ./vendor/bin/pscss -f compressed themes/SuiteP/css/Dawn/style.scss > themes/SuiteP/css/Dawn/style.css
 [Exec] Done in 31.527s
➜  Found style.css for Day, Removing
➜  Found style.scss for Day, Compiling
 [Exec] Running ./vendor/bin/pscss -f compressed themes/SuiteP/css/Day/style.scss > themes/SuiteP/css/Day/style.css
 [Exec] Done in 31.23
```

#### Expected Behavior

To build one color_scheme ;-)

#### Actual Behavior

Builds all color_schemes

#### Possible Fix

Patch function `buildSuitep()` in `lib/Robo/Plugin/Commands/BuildCommands.php`

#### Your Environment

* SuiteCRM Version used: 7.10.20
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 68
* Environment name and version (e.g. MySQL, PHP 7): PHP 7.3
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 19.04

### PR DESCRIPTION
**Pull request by Abuelodelanada**
## Description

Read #7781

## Motivation and Context

Read #7781

## How To Test This

Run:

```bash
$ ./vendor/bin/robo build:suite-p --color_scheme Dawn 
```
And verify that only Dawn color_scheme is compiled
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: lib/Robo/Plugin/Commands/BuildCommands.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ class BuildCommands extends \Robo\Tasks
     public function buildSuiteP(array $opts = ['color_scheme' => ''])
```
Comment (connorshea) [inline lib/Robo/Plugin/Commands/BuildCommands.php]: While we're at it, can we change `color_scheme` to `color-scheme` ;)

Comment (Abuelodelanada) [reply lib/Robo/Plugin/Commands/BuildCommands.php]: Your wishes are orders :-P

Comment (connorshea) [reply lib/Robo/Plugin/Commands/BuildCommands.php]: Heh, I think the phrase in English is usually "Your wish is my command" :)

File: lib/Robo/Plugin/Commands/BuildCommands.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ class BuildCommands extends \Robo\Tasks
      * color_scheme - set which color scheme you wish to build
      * @throws \RuntimeException
      */
-    public function buildSuiteP(array $opts = ['color_scheme' => ''])
+    public function buildSuiteP(array $opts = ['color-scheme' => ''])
```
Comment (connorshea) [inline lib/Robo/Plugin/Commands/BuildCommands.php]: You need to change it for the rest of the file as well :P

Comment (Abuelodelanada) [reply lib/Robo/Plugin/Commands/BuildCommands.php]: It's too Friday afternoon 🤦🏼‍♂

File: lib/Robo/Plugin/Commands/BuildCommands.php (hunk 1 above)
Comment (Abuelodelanada) [reply lib/Robo/Plugin/Commands/BuildCommands.php]: You know that I thought in writing like this!!!
The problem of not trusting in my years of English classes and think that Google translate will do it better than me.

**Comment by Mac-Rae** [comment]
Assessed :+1:

**Pull request merged**